qipr_approver/approver/views/index.py

```diff
@@ -5,12 +5,12 @@
 from django.contrib.auth.models import User
 from approver.models import Project
 
-import json
-import approver.utils as utils
 import approver.constants as constants
+import approver.utils as utils
+from approver.views.dashboard import dashboard
 
 def index(request):
-    context = {
-        'content': 'approver/index.html',
-    }
-    return utils.layout_render(request, context)
+    if(utils.get_current_user_gatorlink(request.session)):
+        return dashboard(request)
```

Comment:
So, based on what I've read and see, the only thing being passed around in the cookie is the sessionid. Django takes this and finds that data in the database with it, which it then assigns to the session variable in the application. It doesn't seem possible to pass alter session variables through http requests.